packages/services/document-service/src/main/resources/application.properties

```diff
@@ -18,3 +18,8 @@ quarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:22.3.0-jav
 ########################################################################################################################
 xdd-document-service/mp-rest/url=https://xdd.wisc.edu
 xdd-extraction-service/mp-rest/url=https://xdddev.chtc.io
+########################################################################################################################
+# Caching (these names correspond to names defined in their java classes - don't change)
+########################################################################################################################
+quarkus.cache.caffeine."xDD-Dictionaries".expire-after-write=86400
```

Comment:
See my comment above on how to manage this.  This will likely become fragile over time.  Additionally, we'll likely want a fixed interval to clear all data caches as having each result expire after a given amount of time could lead to incorrect results.  
eg/ sets is expired and returns updated values and a subsequent call to documents returns a cached value.

Reply:
We should set caching to only an hour for now until we think it's a viable solution.
Or maybe this could be an ENV variable so it's different between `local`, `staging`, and `production`.